{{ cookiecutter.repo_name }}/docsrc/conf.py

```diff
@@ -74,7 +79,7 @@
 
 # If true, the current module name will be prepended to all description
 # unit titles (such as .. function::).
-# add_module_names = True
+# add_package_names = True
```

Comment:
Is this an accidental victim of find and replace?

Reply:
Looks it...Change this and this seems good to go. :shipit:

Reply:
Damn nice catch